modules/nodes-edge-remote-workers-strategies.adoc

```diff
@@ -112,30 +112,28 @@ spec:
 <3> Specify the frequency that the kubelet reports the status of a node associated with this `MachineConfig` object. The default value is `1m`.
 
-The `node-status-update-frequency` parameter works with the `node-monitor-grace-period` and `pod-eviction-timeout` parameters.
+The `node-status-update-frequency` parameter works with the `node-monitor-grace-period` parameter.
 
 * The `node-monitor-grace-period` parameter specifies how long {product-title} waits after a node associated with a `MachineConfig` object is marked `Unhealthy` if the controller manager does not receive the node heartbeat. Workloads on the node continue to run after this time. If the remote worker node rejoins the cluster after `node-monitor-grace-period` expires, pods continue to run. New pods can be scheduled to that node. The `node-monitor-grace-period` interval is `40s`. The `node-status-update-frequency` value must be lower than the `node-monitor-grace-period` value.
 
-* The `pod-eviction-timeout` parameter specifies the amount of time {product-title} waits after marking a node that is associated with a `MachineConfig` object as `Unreachable` to start marking pods for eviction. Evicted pods are rescheduled on other nodes. If the remote worker node rejoins the cluster after `pod-eviction-timeout` expires, the pods running on the remote worker node are terminated because the node controller has evicted the pods on-premise. Pods can then be rescheduled to that node. The `pod-eviction-timeout` interval is `5m0s`.
-
 [NOTE]
 ====
-Modifying the `node-monitor-grace-period` and `pod-eviction-timeout` parameters is not supported.
+Modifying the `node-monitor-grace-period` parameter is not supported.
 ====
 
 --
 
 [id="nodes-edge-remote-workers-strategies-tolerations_{context}"]
 Tolerations::
-You can use pod tolerations to mitigate the effects if the on-premise node controller adds a `node.kubernetes.io/unreachable` taint with a `NoExecute` effect to a node it cannot reach.
+You can use pod tolerations to mitigate the effects if the on-premise node controller adds a `node.kubernetes.io/unreachable` taint with a `NoExecute` effect to a node it cannot reach. 
 
 A taint with the `NoExecute` effect affects pods that are running on the node in the following ways:
 
 * Pods that do not tolerate the taint are queued for eviction.
 * Pods that tolerate the taint without specifying a `tolerationSeconds` value in their toleration specification remain bound forever.
-* Pods that tolerate the taint with a specified `tolerationSeconds` value remain bound for the specified amount of time.  After the time elapses, the pods are queued for eviction.
+* Pods that tolerate the taint with a specified `tolerationSeconds` value remain bound for the specified amount of time. After the time elapses, the pods are queued for eviction. Kubernetes sets the default `tolerationSeconds``, however you can set a preferred value. 
```

Comment:
It does not just set the tolerationSeconds. It adds full `node.kubernetes.io/not-ready` and `node.kubernetes.io/unreachable` tolerations by default. See `Note` sections in https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions

With this in mind, it is probably better to mention the defaulting in the first point or make another point out of it?